teacher.rb

```diff
@@ -0,0 +1,11 @@
+require_relative './person'
+
+class Teacher < Person
+  def initailize(specialization)
+    @specialization = specialization
```

Comment:
* Kindly add super to your class so that you can use all methods from the inherited Person class, for example something like this  :

 ```
require "./animal.rb"

class Dog < Animal
  def initialize(color, name = "Unknown")
    super("dog", 4, name)
    @color = color
  end

  def bring_a_stick
    "Here is your stick: ---------"
  end
end
```